<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillingWindow</class>
 <widget class="QWidget" name="BillingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>Expensas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_date0">
           <property name="text">
            <string>Fecha de Facturación</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QDateEdit" name="dateedit_date">
             <property name="displayFormat">
              <string>dd-MM-yy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_date">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_type">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combobox_type"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_value">
           <property name="text">
            <string>Valor</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineedit_value"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_detail">
           <property name="text">
            <string>Detalle</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_detail"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_voucher">
           <property name="text">
            <string>Comprobante Inicial</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineedit_voucher">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkbox_voucher">
             <property name="text">
              <string>Usar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="button_savegeneralbilling">
         <property name="text">
          <string>Generar Facturación General</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_penalties">
      <attribute name="title">
       <string>Multas por mora</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_since0">
           <property name="text">
            <string>Buscar desde</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_at0">
           <property name="text">
            <string>Registrar con fecha</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDateEdit" name="dateedit_since">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2014</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="displayFormat">
              <string>dd-MM-yy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_since">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QDateEdit" name="dateedit_at">
             <property name="displayFormat">
              <string>dd-MM-yy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_at">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_parms">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table_penalties">
         <property name="columnCount">
          <number>4</number>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="button_generate">
           <property name="text">
            <string>Generar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_savepenalties">
           <property name="text">
            <string>Guardar registros</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_particular">
      <attribute name="title">
       <string>Otros cobros</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_p_date0">
           <property name="text">
            <string>Fecha</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_p_type">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combobox_p_type"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_p_value">
           <property name="text">
            <string>Valor</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_p_value"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_p_detail">
           <property name="text">
            <string>Detalle</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineedit_p_detail"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_p_voucher">
           <property name="text">
            <string>Combrobante</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineedit_p_voucher"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_p_home">
           <property name="text">
            <string>Casa</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="combobox_p_home"/>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QDateEdit" name="dateedit_p_date">
             <property name="displayFormat">
              <string>dd-MM-yy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_p_date">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="button_saveparticularbilling">
         <property name="text">
          <string>Guardar Registro</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
